packages/buefy-next/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "buefy-next",
+  "name": "@ntohq/buefy-next",
```

Comment:
I am pretty sure there is a reason I didn't prefix the package  with `@ntohq` but I can't remember why. I believe it is because the GitHub Action automatically publishes the package to `@ntohq` and if you add this in the `package.json` the script may publish `buefy-next` as `@ntohq/@ntohq/buefy-next`